playbooks/collect_activity/collect_activity.yml

```diff
@@ -15,13 +15,13 @@
       copy:
         src: stats.py
         dest: /tmp/collect_activity.py
-      become_user: www-data
+      become: yes
```

Comment:
So you are running the collect_activity script as root now?  Why is this necessary?

@bdero Would you have any concerns about this change?